docs/compute-engine/end_to_end.ipynb

```diff
@@ -43,7 +24,7 @@
    },
    "outputs": [],
    "source": [
-    "!pip install larq larq-zoo larq-compute-engine"
+    "pip install larq larq-zoo larq-compute-engine"
```

Comment:
Isn't the exclamation mark necessary on colab as well?

Reply:
[Yes, I think so](https://stackoverflow.com/a/51342586).